e2e/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../tsconfig.json",
   "include": ["**/*"],
+  "exclude": [],
```

Comment:
It includes all files inside its root, which is now `e2e`.
However the root `tsconfig` excludes `e2e/**/*`.
So we need to reset the excludes array 🤷🏻‍♂️

Reply:
Little odd, but yeah fine... 😏

Reply:
They behave as two different workspaces though, so if you are in `~/castor` TS will see all files except `e2e`.
If you `cd` into `~/castor/e2e` however, TS will only see files inside that folder, and nothing else.
The config is a bit odd I suppose, because we want to reuse most of the definitions, but the result makes sense I think.